charts/synthetics-private-location/templates/deployment.yaml

```diff
@@ -24,6 +24,9 @@ spec:
       {{ if .Values.dnsPolicy }}
       dnsPolicy: {{ .Values.dnsPolicy}}
       {{ end }}
+      {{ if .Values.dnsConfig }}
+      dnsConfig: {{ .Values.dnsConfig}}
```

Comment:
I believe this should follow the same structure as the other `object` type values like [affinity](https://github.com/DataDog/helm-charts/blob/efa7e1f894ea127612b1fe22242ba5277249fe62/charts/synthetics-private-location/templates/deployment.yaml#L93-L94) a little further down this file. I think you just need to switch the `if` to `with`, put it on the next line, and add a `toYaml` and `indent`. Could you also include a space between the end of `dnsConfig` and the trailing `}}` please?

Reply:
Thank you; all done